### ISSUE TITLE
Unclear jump in inference time

### ISSUE THREAD

**Issue by DdHms** [tensorflow/tfjs#7180]
I am converting a keras U-net standard model to tfjs using the function `tfjs.converters.save_keras_model(MODEL, TFJS_PATH)
`.
I noticed that the first concatenation with the skip connection not from the nearly deepest encoder output adds a significant inference time:
this model only takes 60 ms to run after conversion:
```
x_in = keras.layers.Input(shape=(224, 224, 3))
e_stage_1, e_stage_2, e_stage_3, e_stage_4, _, e_stage_5, _, e_stage_6 = EfficientNetLiteB0(weights=None, input_shape=(224, 224, 3), include_top=False)(x_in)
"""-------------Bridge--------------"""

bridge = conv_block(e_stage_6, B, dilation=2)

"""-------------Decoder--------------"""
#
# "decoder stage 6"
# # dilation difference
x = layers.concatenate([e_stage_6, bridge], axis=-1)
d_stage_6 = conv_block(x, D_G)
#
x = layers.UpSampling2D(size=(2, 2))(d_stage_6)

model = keras.models.Model(inputs=[x_in],
                     outputs=[x])
```
adding one more stage to the decoder adds another 40ms:
```
stage_1, e_stage_2, e_stage_3, e_stage_4, _, e_stage_5, _, e_stage_6 = EfficientNetLiteB0(weights=None, input_shape=(224, 224, 3), include_top=False)(x_in)
"""-------------Bridge--------------"""

bridge = conv_block(e_stage_6, B, dilation=2)

"""-------------Decoder--------------"""
#
# "decoder stage 6"
# # dilation difference
x = layers.concatenate([e_stage_6, bridge], axis=-1)
d_stage_6 = conv_block(x, D_G)
#
# "decoder stage 5"
#  1/32 --> 1/16
#
x = layers.UpSampling2D(size=(2, 2))(d_stage_6)
x = layers.concatenate([e_stage_5, x], axis=-1)
d_stage_5 = conv_block(x, D_F)
model = keras.models.Model(inputs=[x_in], outputs=[x])
```
whereas replacing this line: `x = layers.concatenate([e_stage_5, x], axis=-1)`  with this: `x = layers.concatenate([x, x], axis=-1)`
added no significant time even though the number of filters in x is greater than that of e_stage_5.

How does this make sense and what could be done?

**Comment by pyu10055** [comment]
@DdHms thank you for reporting this, can you create a demo in codepen or other place you prefer, helping us to reproduce this issue? thanks

**Comment by gaikwadrahul8** [comment]
Hi, @DdHms 

Apologize for the delayed response and thank you for bringing this issue to our attention, Could you please help us with your code-snippet or colab file where you're trying to convert your keras model into `Tensorflow.js` so we'll try to replicate the same issue from our end 

Please refer these official documentation references meanwhile [Ref-1](https://www.tensorflow.org/js/tutorials/conversion/import_keras), [Ref-2](https://github.com/tensorflow/tfjs/tree/master/tfjs-converter). Thank you!
